recipe/meta.yaml

```diff
@@ -49,13 +48,12 @@ requirements:
     - r-tmvtnorm
     # for compatibility with r-rcpparmadillo
     - llvm-openmp  # [osx]
-    - libgomp      # [linux]
+    - libgomp      # [not osx]
```

Comment:
Would move both of these to `requirements/build` and drop them from other sections. As both of them have `run_exports/strong`, they will automatically be added to all other `requirements/*` sections